esterni/piano-di-qualifica/components/report-verifica.tex

```diff
@@ -0,0 +1,78 @@
+\documentclass[../piano-di-qualifica.tex]{subfiles}
+
+
+\begin{document}
+  \subsection{Analisi statica dei documenti}%
+  \label{sub:analisi_statica_doc}
+	Per l'analisi statica dei documenti GruppOne ha utilizzato il package \glossario{chktex} che ha consentito di trovare errori nella sintassi latex.
+  Il test, eseguito ad ogni modifica repository, ha dato esito positivo per ogni documento segnalando complessivamente 0 errori.
+
+  \begin{figure}[H]
+    \centering
+    \includegraphics[width=160mm]{correttezzaortografica-RR.png}%
+  \end{figure}
+
+
+  \begin{figure}[H]
+    \centering
+    \includegraphics[width=160mm]{correttezzaortografica-RP.png}%
+  \end{figure}
+
+
+
+	% sub:analisi_statica_doc (end)
+  \subsection{Esito verifica leggibilità}%
+  \label{sub:verifica_leggibilita}
+	Allo scopo di assicurarci una leggibilità accettabile dei documenti abbiamo monitorato l'indice \glossario{Gulpease} di ogni documento. Qui sotto riportiamo i risultati della versione consegnata in RR e successive relativamente agli obiettivi di qualità che erano stati definiti.
```

Comment:
"i risultati associati alle versioni consegnate in occasione delle revisioni di avanzamento" invece di "i risultati della versione consegnata in RR e successive"